alb_template/variables.tf

```diff
@@ -35,6 +35,12 @@ variable "acm_certificate_alternative_names" {
   description = "Alternative names of the SSL certificate. If you use CNAME for the domain_name, you should set FQDN of CNAMEs to allow access from the CNAMEs."
 }
 
+variable "ssl_policy" {
+  type        = string
+  default     = "ELBSecurityPolicy-2016-08"
```

Comment:
Is it okay to make `""` as default?

If so, it seems to be better rather than hard-cording `ELBSecurityPolicy-2016-08` (only for current HTTPS).

Reply:
I fixed it. Thank you.